ansible-ipi-install/README.md

```diff
@@ -1,29 +1,29 @@
-Table of contents
-=================
+# Table of contents
 
 <!--ts-->
-   * [Introduction](#introduction)
-   * [Prerequisites](#prerequisites)
-   * [Tour of the Ansible Playbook](#tour-of-the-ansible-playbook)
-   * [Running the Ansible Playbook](#running-the-ansible-playbook)
-      * [The `ansible.cfg` file](#the-ansiblecfg-file)
-      * [Copy local ssh key to provision node](#copy-local-ssh-key-to-provision-node)
-      * [Modifying the `inventory/hosts` file](#modifying-the-inventoryhosts-file)
-      * [The Ansible `playbook.yml`](#the-ansible-playbookyml)
-      * [Adding Extra Configurations to the OpenShift Installer](#adding-extra-configurations-to-the-openshift-installer)
-      * [Pre-caching RHCOS Images](#pre-caching-rhcos-images)
-      * [Disconnected Registry](#disconnected-registry)
-      * [Running the `playbook.yml`](#running-the-playbookyml)
-   * [Verifying Installation](#verifying-installation)
-   * [Troubleshooting](#troubleshooting)
-      * [Unreachable Host](#unreachable-host)
-      * [Permission Denied Trying To Connect To Host](#permission-denied-trying-to-connect-to-host)
-      * [Dig lookup requires the python 'dnspython' library and it is not installed](#dig-lookup-requires-the-python-dnspython-library-and-it-is-not-installed)
-   * [Gotchas](#gotchas)
-   * [Appendix A](#appendix-a-using-ansible-tags-with-the-playbookyml)
-      * [How to use the Ansible tags](#how-to-use-the-ansible-tags)
-      * [Skipping particular tasks using Ansible tags](#skipping-particular-tasks-using-ansible-tags)
-<!--te-->
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Tour of the Ansible Playbook](#tour-of-the-ansible-playbook)
+- [Running the Ansible Playbook](#running-the-ansible-playbook)
+  - [The `ansible.cfg` file](#the-ansiblecfg-file)
+  - [Copy local ssh key to provision node](#copy-local-ssh-key-to-provision-node)
+  - [Modifying the `inventory/hosts` file](#modifying-the-inventoryhosts-file)
+  - [The Ansible `playbook.yml`](#the-ansible-playbookyml)
+  - [Adding Extra Configurations to the OpenShift Installer](#adding-extra-configurations-to-the-openshift-installer)
+  - [Pre-caching RHCOS Images](#pre-caching-rhcos-images)
+  - [Disconnected Registry](#disconnected-registry)
+  - [Running the `playbook.yml`](#running-the-playbookyml)
+- [Verifying Installation](#verifying-installation)
+- [Troubleshooting](#troubleshooting)
+  - [Unreachable Host](#unreachable-host)
+  - [Permission Denied Trying To Connect To Host](#permission-denied-trying-to-connect-to-host)
+  - [Dig lookup requires the python 'dnspython' library and it is not installed](#dig-lookup-requires-the-python-dnspython-library-and-it-is-not-installed)
+- [Gotchas](#gotchas)
+- [Appendix A](#appendix-a-using-ansible-tags-with-the-playbookyml)
+  _ [How to use the Ansible tags](#how-to-use-the-ansible-tags)
+  _ [Skipping particular tasks using Ansible tags](#skipping-particular-tasks-using-ansible-tags)
```

Comment:
Should these be `-` instead of `_` ? Looks odd in the preview. And thank you so much for the fixes!

Reply:
Hi Roger,
I was testing the pre-commit hook we used for community that automatically 'fixes' markdown, python formatting (via black), shell format (via shfmt).

Ignore this in the meantime :-)

Thanks!

Reply:
@rlopez133 can you review it now? 

Thanks!